templates/utils.js

```diff
@@ -26,4 +26,10 @@ export const withDefaults =
     return template(argsWithDefault);
   };
 
-export const stringify = val => inspect(val, { depth: null, compact: true, maxArrayLength: null, maxStringLength: null })
+export const stringify = val => {
+  const str = inspect(val, { depth: null, compact: true, maxArrayLength: null, maxStringLength: null });
+  return str
+    .replace(/"\$\$\$([^"]+)"/g, '$1')
+    .replace(/'\$\$\$([^']+)'/g, '$1')
```

Comment:
these two lines replaces $$$var with var, so if we meet {a: "$$$var"} or {a: '$$$var'} in code it results {a: var}. Also works for {a: '$$$process.env.B = "true"'} => {a: process.env.B = "true"}